.github/workflows/main.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - uses: Swatinem/rust-cache@9d47c6ad4b02e050fd481d890b2ea34778fd09d6 # v2.7.8
 
       - name: Install Zola
-        run: cargo install --locked --git https://github.com/senekor/zola --rev 79410eea82f837e4de9b1e4c3905287060b69255
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/senekor/zola/releases/download/v0.20.1-senekor/zola-installer.sh | sh
```

Comment:
hm, not a fan of doing `curl | sh` from an unofficial URL. `cargo install` with a git hash is a bit more reliable.

For CI we can use GHA caching (i think the cargo-install action can do it automatically now?). For devs I'm okay with the `cargo install`.

Reply:
Yeah, that makens sense. Reverted it to just update the commit hash. I copied the install command from the readme without thinking much. We already have `Swatinem/rust-cache` set up, that takes care of the caching. So there's no downside to building from source, like you said.